composer.json

```diff
@@ -61,7 +63,9 @@
     },
     "conflict": {
         "doctrine/dbal": "<2.12.0",
-        "doctrine/orm": "<2.6.3"
+        "doctrine/orm": "<2.6.3",
+        "doctrine/doctrine-module": "<4.0",
+        "doctrine/doctrine-orm-module": "<4.0"
```

Comment:
Can we run `maglnet/composer-require-checker` on the codebase, to discover if any of these conflicts are part of symbols that we use?

If they are, we should just have them in `require` instead.

Reply:
```
php composer-require-checker.phar check composer.json
ComposerRequireChecker 4.0.0@baa11a4e9e5072117e3d180ef16c07036cafa4a2
The following 51 unknown symbols were found:
+-----------------------------------------------------------+--------------------+
| Unknown Symbol                                            | Guessed Dependency |
+-----------------------------------------------------------+--------------------+
| DoctrineModule\Persistence\ObjectManagerAwareInterface    |                    |
| DoctrineModule\Validator\NoObjectExists                   |                    |
| DoctrineModule\Validator\ObjectExists                     |                    |
| Doctrine\Common\Collections\ArrayCollection               |                    |
| Doctrine\Laminas\Hydrator\DoctrineObject                  |                    |
| Doctrine\ORM\EntityManager                                |                    |
| Doctrine\ORM\EntityManagerInterface                       |                    |
| Doctrine\ORM\NoResultException                            |                    |
| Doctrine\ORM\QueryBuilder                                 |                    |
| Doctrine\ORM\Tools\Pagination\Paginator                   |                    |
| Doctrine\Persistence\ObjectManager                        |                    |
| interop\container\containerinterface                      |                    |
| Laminas\ApiTools\Admin\Exception\InvalidArgumentException |                    |
| Laminas\ApiTools\Admin\Exception\RuntimeException         |                    |
| Laminas\ApiTools\Admin\Model\AbstractAutodiscoveryModel   |                    |
| Laminas\ApiTools\Admin\Model\DocumentationModel           |                    |
| Laminas\ApiTools\Admin\Model\InputFilterModel             |                    |
| Laminas\ApiTools\Admin\Model\ModuleEntity                 |                    |
| Laminas\ApiTools\Admin\Model\ModuleModel                  |                    |
| Laminas\ApiTools\Admin\Model\ModulePathSpec               |                    |
| Laminas\ApiTools\Admin\Model\NewRestServiceEntity         |                    |
| Laminas\ApiTools\Admin\Model\RestServiceEntity            |                    |
| Laminas\ApiTools\Admin\Model\RestServiceModelFactory      |                    |
| Laminas\ApiTools\Admin\Model\RestServiceResource          |                    |
| Laminas\ApiTools\Admin\Model\RpcServiceEntity             |                    |
| Laminas\ApiTools\Admin\Model\RpcServiceModelFactory       |                    |
| Laminas\ApiTools\Admin\Model\RpcServiceResource           |                    |
| Laminas\ApiTools\Admin\Utility                            |                    |
| Laminas\ApiTools\ApiProblem\ApiProblem                    |                    |
| Laminas\ApiTools\Configuration\ConfigResource             |                    |
| Laminas\ApiTools\Configuration\ConfigResourceFactory      |                    |
| Laminas\ApiTools\ContentNegotiation\ViewModel             |                    |
| Laminas\ApiTools\Doctrine\Admin\Model\InvokableFactory    |                    |
| Laminas\EventManager\Event                                |                    |
| Laminas\EventManager\EventManager                         |                    |
| Laminas\EventManager\EventManagerAwareInterface           |                    |
| Laminas\EventManager\EventManagerInterface                |                    |
| Laminas\EventManager\ListenerAggregateInterface           |                    |
| Laminas\EventManager\SharedEventManager                   |                    |
| Laminas\EventManager\SharedEventManagerInterface          |                    |
| Laminas\Filter\FilterChain                                |                    |
| Laminas\Filter\StringToLower                              |                    |
| Laminas\Filter\Word\CamelCaseToDash                       |                    |
| Laminas\InputFilter\CollectionInputFilter                 |                    |
| Laminas\InputFilter\InputFilterInterface                  |                    |
| Laminas\ModuleManager\ModuleManager                       |                    |
| Laminas\Mvc\Controller\AbstractActionController           |                    |
| Laminas\Mvc\Controller\ControllerManager                  |                    |
| Laminas\Mvc\ModuleRouteListener                           |                    |
| Laminas\Mvc\Service\AbstractPluginManagerFactory          |                    |
| Laminas\Paginator\Adapter\AdapterInterface                |                    |
+-----------------------------------------------------------+--------------------+
```

Reply:
That is **a lot** of stuff @_@ 

I don't think we can add all these in this patch though...

Reply:
I think we can fix these here by just adding the `require/` blocks in the code, but I'd be OK for us doing this in `3.1.x` later